### ISSUE TITLE
Idea: subdivide cells

### ISSUE THREAD

**Issue by mojodna** [fieldpapers/fieldpapers#234]
While in Ecuador, we realized that we wanted to subdivide certain cells with a high density of buildings for a couple reasons: to more evenly split up the work of enumerating those features and to produce more detailed maps for areas that specifically needed it.

A potential user interface would allow individual grid cells (/ atlas pages) to be selected and optionally split (presumably in 4).

**Comment by aawiseman** [comment]
Yes, definitely!

**Comment by wonderchook** [comment]
Yes this is great. You probably want to think through the ability to sub-divide but then be able to merge. Or at least limit the amount of subdivisions, the Tasking Manager allows such divisions but we found that some people went a little overboard on this.
